### DISCUSSION TITLE
cds-pg → CAP community core

### DISCUSSION THREAD

**Discussion by vobu** [sapmentors/cds-pg#363] [General]
as many of you have noticed, there currently is no support for `cds^6`. This is mostely due to a ping-pong of ideas how to bring `cds-pg` closer to the CAP core. 
With the announcement of parts of CAP becoming Open Source throughout 2023, this is now happening. 
Although we can't provide specific dates, **`cds-pg` will move under the umbrella of Open Source CAP**. From then on, it will be co-maintained by the SAP CAP core team _together_ with the CAP community - including official support. 
We'll use this opportunity to also hook up `cds-pg` to the renovated CAP core database layer. This means that it will probably not support `cds^6`, but for sure `cds^7`. 
Again, we were asked to not communicate a timeline. But please watch the CAP-related announcements at TechEd next week. Things will happen rather sooner than later 😉😎
(ping to #355 #285 #343)

**Comment by remi-sap** [comment]
Any good news to share?

**Reply by vobu** [reply]
yes! work has started, see https://twitter.com/vobu/status/1621494806575816704
expect an alpha in March 😄

**Comment by vobu** [comment]
meanwhile, @ryegrosT8 has upped the game and put in cds^6 support! 👏 🙌 
https://github.com/sapmentors/cds-pg/pull/383

**Comment by GoWale** [comment]
Hi,
I'm just happy that I found this glimpse of hope.
As an ABAP developer, that started end of last year with all this SAP BTP things, lets say the struggle was there.
Starting on the BTP will be much easier and less discussion when there will be a small DB solution for the first litte Applications.

Tanks for all the time and afford you have put in this project.
Norbert

**Reply by gregorwolf** [reply]
Hi Norbert,

with the SAP HANA Cloud included in BTP Free Tier there is at least a good option for development. And when you follow the SAP TechEd 2022 Hands-on: [XP261 - Operating Your SAP BTP Platform Environment with SAP Cloud ALM](https://github.com/SAP-samples/teched2022-XP261) you learn how the HANA can be re-started automatically.

Best Regards
Gregor

**Reply by GoWale** [reply]
Hi Gregor,
for the development I really love how generous the market place is. (Customers and Industries are different and I want to leave it there)
And thanks for the tipp for the ALM. Time to see when "on a nightly basis" is in each region.
regards
Norbert
